modules/demo-px/tailwind.config.js

```diff
@@ -1,13 +1,32 @@
 import colors from 'tailwindcss/colors.js'
 import forms from '@tailwindcss/forms'
+import { readFile } from 'node:fs/promises'
+import { argv } from 'node:process'
+
+const getTailwindContent = async (source) => {
```

Comment:
I dont like this being duplicated. Move it to theme-tailwind module (maybe index.js in the root of the module) and import it here via `@storefront-x/theme-tailwind` import name (ioc wont work here)